### DISCUSSION TITLE
New icon for Input-Leap

### DISCUSSION THREAD

**Discussion by Cimbali** [input-leap/input-leap#1598] [Ideas]
With input-leap now being a separate project from barrier, it could be good to change the icon, in addition to the name.

I’ve browsed some icon websites, with free icons (under CC license, or alternately can be bought royalty-free for ~3$), and here are a few suggestions (purely as conversation starter, of course):

There’s a variety of leaping animals;
- [<img src="https://user-images.githubusercontent.com/6126377/218317736-2262a4f8-eaa5-4b1f-a218-b8b3e058b244.png" width="100">](https://thenounproject.com/icon/goat-2995016/)
- [<img src="https://user-images.githubusercontent.com/6126377/218317893-385fad4f-b5fc-4a71-acd4-1096f994cc2b.png" width="100">](https://thenounproject.com/icon/cow-2995017/)
- [<img src="https://user-images.githubusercontent.com/6126377/218317868-91d9f07d-5242-44bb-bf85-5d8d7819bde1.png" width="100">](https://thenounproject.com/icon/sheep-2995015/)
- [<img src="https://user-images.githubusercontent.com/6126377/218317898-3de6f432-7aac-425e-a2ef-ea1b7e0e4105.png" width="100">](https://thenounproject.com/icon/kangaroo-4742/)
- [<img src="https://user-images.githubusercontent.com/6126377/218317882-464d61dd-a779-456e-920b-84b909ee020c.png" width="100">](https://thenounproject.com/icon/fish-1439542/)

Some more conceptual representations of a “leap”
- [<img src="https://user-images.githubusercontent.com/6126377/218318001-d87e669c-e2c9-4273-acfb-601e955550ed.png" width="100">](https://thenounproject.com/icon/jump-2623561/)
- [<img src="https://user-images.githubusercontent.com/6126377/218318016-3cb18d78-9ed6-44a5-a87e-a5b6886e62d0.png" width="100">](https://thenounproject.com/icon/overtake-915991/)

Some options that are more related to the notion of input:
- [<img src="https://user-images.githubusercontent.com/6126377/218318474-607766cd-e8a9-41e2-81ab-a4cf5b496d32.png" width="100">](https://thenounproject.com/icon/input-4886685/)
- [<img src="https://user-images.githubusercontent.com/6126377/218318500-55d198a2-025a-4146-89c5-fe4d109e488e.png" width="100">](https://thenounproject.com/icon/input-3330265/)
- [<img src="https://user-images.githubusercontent.com/6126377/218318524-d0effb6c-78c2-4e59-9a4a-331058cde4e6.png" width="100">](https://thenounproject.com/icon/input-1550854/)
- [<img src="https://user-images.githubusercontent.com/6126377/218318540-6d9fbae9-15ff-4eca-94f0-534f60c6ea8f.png" width="100">](https://thenounproject.com/icon/input-1066840/)

And a few directly displaying keyboard/mouse
- [<img src="https://user-images.githubusercontent.com/6126377/218318734-c8266e43-a128-4f4a-8b2f-72020b30332e.png" width="100">](https://thenounproject.com/icon/keyboard-mouse-1931207/)
- [<img src="https://user-images.githubusercontent.com/6126377/218318570-6e49136b-c7a0-413c-9ccb-0b10939afebe.png" width="100">](https://thenounproject.com/icon/input-693935/)
- [<img src="https://user-images.githubusercontent.com/6126377/218318576-7769bac0-af3f-4360-abf8-e4eaf497ba84.png" width="100">](https://thenounproject.com/icon/input-device-1508362/)
- [<img src="https://user-images.githubusercontent.com/6126377/218318595-955bc975-cf9e-449e-9270-1e1573238cac.png" width="100">](https://thenounproject.com/icon/mouse-keyboard-4432054/)

Of course these can also be combined and coloured to convey e.g. connection status (though that implies royalty-free option)

<img width="100" alt="input-leap-green" src="https://user-images.githubusercontent.com/6126377/221367763-638e7614-589c-47b9-86e6-0947fff7b6b9.png" /> <img width="100" alt="input-leap-orange" src="https://user-images.githubusercontent.com/6126377/221367764-b9992102-1e4a-45f3-b1cf-85964e178d28.png" /> <img width="100" alt="input-leap-red" src="https://user-images.githubusercontent.com/6126377/221367765-bfdc54ca-4611-4fcf-b862-caa9924ca61c.png" />

**Comment by shymega** [comment]
@p12tic has found a good icon, I think we've settled on something else. I'll see if they can make the PR or post it here for voting.
